lang/py/src/avro/io.py

```diff
@@ -38,11 +38,13 @@
 """
 import struct
 from avro import schema
+from avro import constants
 import sys
 from binascii import crc32
 
 from decimal import Decimal
 from decimal import getcontext
+import datetime
```

Comment:
This is not critical, but it's nicer to group the imports by python packages, third-party packages, and modules in this repo.